.github/workflows/upgrade-develop-testing.yml

```
@@ -67,7 +67,7 @@ jobs:
         db-type: [ 'mysql' ]
         db-version: [ '5.7', '8.4' ]
         # WordPress 4.9 is the oldest version that supports PHP 7.2.
-        wp: [ '4.9', '6.7', '6.8', '6.9-RC1' ]
+        wp: [ '4.9', '6.7', '6.8', '6.9' ]
```

```suggestion
        wp: [ '4.9', '6.8', '6.9' ]
```
I believe that the intention here was to test the currently supported branch +1 previous. @johnbillion can you confirm that?

We should add a [docblock similar to the one in `upgrade-testing.yml` explaining the ideal strategy matrix](https://github.com/WordPress/wordpress-develop/blob/trunk/.github/workflows/upgrade-testing.yml#L40-L58) for our future selves.